projects/lodepng/lodepng_fuzzer.dict

```diff
@@ -0,0 +1,39 @@
+#
```

Comment:
Remove this and directly add from https://github.com/google/AFL/blob/master/dictionaries/png.dict in Dockerfile.

Reply:
Sure, but if that link is broken or they decided to change the location, we would have to do the same so copied it. I guess a direct link to gcs bucket or the original location of that dictionary would be better, but I was not able to find that.

Reply:
Use https://raw.githubusercontent.com/google/AFL/master/dictionaries/png.dict
Link should not change. We always use the source of truth and avoid copying duplicates in OSS-Fuzz repo.

Reply:
Sure, thanks! I will make the required changes asap.